src/useCookieValue/__docs__/story.mdx

```diff
@@ -64,17 +64,16 @@ import { useCookieValue } from '@react-hookz/web/esnext/useCookieValue/useCookie
 
 #### Arguments
 
-- **key** _`string`_ - Cookie name to manage.
-- **options** _`UseCookieOptions`_ _(default: {})_ - Cookie options that will be used during cookie
-  set and delete. Has only one extra option, that relates to the hook itself:
-  - **initializeWithValue** _`boolean`_ _(default: true)_ - Whether to initialize state with cookie
-    value or initialize with `undefined` state.  
-    _Default to `false` during SSR._
+- **key** _`string`_ - Name of the cookie to manage.
+- **options** _`UseCookieOptions`_ _(default: {})_ - Cookie options that will be used during setting
+    and deleting the cookie. Has one extra option, that relates to the hook itself:
+  - **initializeWithValue** _`boolean`_ _(default: true)_ - Whether to initialize state with the cookie
+    value or `undefined`. _We suggest setting this to `false` during SSR._
```

Comment:
Is the intention that the user needs to set this to false during SSR, or is it that the hook actually defaults the value to false during SSR?

I am reading this as that the hook automatically defaults this to false during SSR.

Reply:
Hmm... The user needs to set this to false during SSR. Do you think it would be clearer to just say something like "Set this to false during SSR"?